### PR TITLE
Prepare release 1.5

### PR DESCRIPTION
**Pull request by gregkalapos**
- [x] bump version
- [x] create changelog

### CODE REVIEW

**Comment by gregkalapos** [comment]
I don't really [see in the logs](https://elasticsearch-ci.elastic.co/job/elastic+docs+apm-agent-dotnet+pull-request/1085/console) why this doc build fails. @bmorelli25 could you please take a look? 

I have some link that are not yet on `current`- can that be the problem? Like [here](https://github.com/elastic/apm-agent-dotnet/pull/840/files#diff-634e7b38da1fcb3d58e9db4eaa5653f2R30) - that's only on `master` right now - my plan was to move that to `current` (aka `1.x` branch) once the 1.5 release is out.

**Comment by bmorelli25** [comment]
This is an interesting one. Is there a reason why you're not using attributes to link to content? That would ensure the release notes in each branch only link to content that exists in the same branch. I'll leave a few comments.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -21,6 +21,22 @@ endif::[]
...
 
+[[release-notes-1.5.0]]
+==== 1.5.0
+
+[float]
+===== Features
+- Auto instrumentation for `SqlClient` (https://www.elastic.co/guide/en/apm/agent/dotnet/current/setup.html#setup-sqlclient[documentation])
+- Introducing Filter API {pull}792[#792] (https://www.elastic.co/guide/en/apm/agent/dotnet/current/public-api.html#filter-api[documentation])
+- Auto-detect culprit for exceptions {pull}740[#740]
+- New config settings: `ExcludedNamespaces`, `ApplicationNamespaces` (https://www.elastic.co/guide/en/apm/agent/dotnet/current/config-all-options-summary.html[documentation])
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:32]:
```suggestion
- Auto instrumentation for `SqlClient` (<<setup-sqlclient,documentation>>)
- Introducing Filter API {pull}792[#792] (<<filter-api,documentation>>)
- Auto-detect culprit for exceptions {pull}740[#740]
- New config settings: `ExcludedNamespaces`, `ApplicationNamespaces` (<<config-all-options-summary,documentation>>)
```

Comment (bmorelli25) [reply CHANGELOG.asciidoc:32]: This should fix all of the documentation errors as all of this content exists in master. When the release is pushed to 1.x, all of the content and links in the 1.x branch will be updated to point to current instead.

The only downside of this is that the links are not clickable in markdown. I think that's ok though, as neither are the PR links, and we have a header at the top that encourages readers to view RNs in the documentaiton.

**Comment by gregkalapos** [comment]
Thanks @bmorelli25! 

> Is there a reason why you're not using attributes to link to content? 

That was lack of knowledge on my side - I did not know it works also from here; but now I see how I should use this 👍 

> The only downside of this is that the links are not clickable in markdown. I think that's ok though, as neither are the PR links, and we have a header at the top that encourages readers to view RNs in the documentation.

Yes, I agree. I'll also do a GitHub release here, and for that I'll turn those into markdown for [this page](https://github.com/elastic/apm-agent-dotnet/releases) - I think that's where people would look at changelogs if they are on GitHub.

**Comment by vhatsura** [comment]
@gregkalapos, what about #794. Are you going to announce it after solving the issue with `LogLevel`?

**Comment by gregkalapos** [comment]
> @gregkalapos, what about #794. Are you going to announce it after solving the issue with `LogLevel`?

That's a feature which also depends on Kibana, so it mainly targets 7.7+ - these additional configs should be covered in the release announcement of 7.7.

Not that I thought about this a lot, but basically 2 reasons I have not listed:
- yeah, it'd be nice to have some idea about the loglevel
- and I felt as long as 7.7 is not officially GA, listing it would be more confusing.

**Pull request merged**